component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -1338,7 +1338,9 @@ private String importUser(UserImportResultEntity userImportResultEntity,
       userImportResultEntity.addErrorMessage(userName, errorMessage);
       return userName;
     }
+    String status = userObject.getString("status").toLowerCase();
```

Comment:
is « status » name of the column in the file? If so why is is status and not active? or enabled ?

Reply:
yes